src/Microsoft.DotNet.Interactive.Tests/Parsing/DirectiveParserTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.FSharp;
+using Microsoft.DotNet.Interactive.Parsing;
+using System.Threading.Tasks;
+using Xunit;
+using System;
+using System.CommandLine;
+
+namespace Microsoft.DotNet.Interactive.Tests.Parsing
+{
+    public class DirectiveParserTests
+    {
+        private static Command GetRDirective()
```

Comment:
`r` is private, so I can make it internal and make the assembly visible to the test. Another option is to test `SubmissionParser`, but it's a bit more complicated and high-level than `DirectiveParser`